cmake/CMakeLists.txt

```diff
@@ -257,6 +261,10 @@ if(BUILD_OMP)
     message(FATAL_ERROR "Cannot find the 'omp.h' header file required for full OpenMP support")
   endif()
   target_link_libraries(lammps PRIVATE OpenMP::OpenMP_CXX)
+else()
+  if(Kokkos_ENABLE_OPENMP)
+    message(FATAL_ERROR "Must enable BUILD_OMP with Kokkos_ENABLE_OPENMP")
```

Comment:
I would move that error message using `if(Kokkos_ENABLE_OPENMP AND NOT BUILD_OMP)` to `KOKKOS.cmake` to make it more self-contained.

Reply:
ok. thanks for the suggestion.